assets/js/components/KeyMetrics/MetricsSelectionPanel/Footer.js

```diff
@@ -212,9 +209,7 @@ export default function Footer( {
 		trackEvent( trackingCategory, 'metrics_sidebar_cancel' );
 	}, [ trackingCategory ] );
 
-	const maxSelectedMetricsLimit = isConversionReportingEnabled
-		? MAX_SELECTED_METRICS_COUNT_WITH_CONVERSION_EVENTS
-		: MAX_SELECTED_METRICS_COUNT;
+	const maxSelectedMetricsLimit = MAX_SELECTED_METRICS_COUNT;
```

Comment:
We don't need this variable anymore, let's use `MAX_SELECTED_METRICS_COUNT` directly similarly how we use `MIN_SELECTED_METRICS_COUNT`.

Reply:
@eugene-manuilov this has been resolved in my latest commits.